x/auth/types/credentials.go

```diff
@@ -62,7 +62,7 @@ func (m *ModuleCredential) VerifySignature(_ []byte, _ []byte) bool {
 }
 
 func (m *ModuleCredential) Equals(other cryptotypes.PubKey) bool {
-	return false
+	return m.String() == other.String()
```

Comment:
Is this the right way to do this?

Reply:
That is _one_ way.
I didn't want to loop through all the fields. However that's the most correct way. Will fix.